osquery/core/posix/process.cpp

```diff
@@ -127,4 +129,20 @@ std::shared_ptr<PlatformProcess> PlatformProcess::launchExtension(
 
   return std::make_shared<PlatformProcess>(ext_pid);
 }
+
+std::shared_ptr<PlatformProcess> PlatformProcess::launchPythonScript(
+    const std::string& args) {
+  std::shared_ptr<PlatformProcess> process;
+
+  int process_pid = ::fork();
+  if (process_pid == 0) {
+    // Start a Python script
+    ::execlp("sh", "sh", "-c", args.c_str(), nullptr);
```

Comment:
For the Python path on POSIX, is `/usr/local/osquery/bin/python` a good assumption?